src/server/pachyderm_test.go

```diff
@@ -5010,6 +5011,53 @@ func TestMaxQueueSize(t *testing.T) {
 	}, backoff.RetryEvery(500*time.Millisecond).For(20*time.Second)))
 }
 
+func TestHTTPGetFile(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	t.Parallel()
+	c := getPachClient(t)
+
+	dataRepo := uniqueString("TestHTTPGetFile_data")
+	require.NoError(t, c.CreateRepo(dataRepo))
+
+	commit1, err := c.StartCommit(dataRepo, "master")
+	require.NoError(t, err)
+	_, err = c.PutFile(dataRepo, commit1.ID, "file", strings.NewReader("foo"))
+	f, err := os.Open("../../etc/testing/artifacts/giphy.gif")
+	require.NoError(t, err)
+	_, err = c.PutFile(dataRepo, commit1.ID, "giphy.gif", f)
+	require.NoError(t, err)
+	require.NoError(t, c.FinishCommit(dataRepo, commit1.ID))
+
+	// Try to get raw contents
+	resp, err := http.Get(fmt.Sprintf("http://127.0.0.1:30652/v1/pfs/repos/%v/commits/%v/files/file", dataRepo, commit1.ID))
```

Comment:
How would you feel about adding a GetAddr() method to client.APIClient and using that, rather than `127.0.0.1:30650`? The downside to having a hard-coded address here is just that it only works with `pc port-forward`, but doesn't work with `ADDRESS` (which incidentally means it doesn't quite work in my testing environment, though that's something I could easily fix).

The downside to `GetAddr()` is it widens the interface to `client.APIClient`—your call